test/run_e2e_test.sh

```diff
@@ -0,0 +1,20 @@
+TAGS=$@
+curdir=`pwd`
+cd `dirname $0`
+
+projectdir=../
+cd $projectdir
+
+# Check Dependencies
+#[[ ! -f func ]] && echo "func binary not found. run 'make build' prior to run e2e." && exit 1
```

Comment:
Why is this commented?

Reply:
yea there is no need to be commented. I probably disabled during some local vs gitactions tests.

Reply:
I'm just marking as resolved since there a new pr for e2e which considers this suggestion